scripts/workers/code_upload_worker_utils/make_submission.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-submission_url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/challenge_phase/$PHASE_PK/submission/$SUBMISSION_PK"
+submission_url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/update_submission/"
```

Comment:
Where is the submission pk for this API call?

Reply:
Updated.